firmware/tunerstudio/rusefi.input

```diff
@@ -937,8 +937,13 @@ menuDialog = main
 
 	menu = "Ignition"
 		subMenu = ignitionSettings,			"Ignition Settings"
+		subMenu = dwellSettings,			"Dwell time"
```

Comment:
We're moving dwell table out of the settings panel to a separate menu item. This helps users to find it in menu.